src/Lean/Elab/BuiltinCommand.lean

```diff
@@ -126,6 +194,26 @@ private partial def elabChoiceAux (cmds : Array Syntax) (i : Nat) : CommandElabM
   | Except.ok env   => setEnv env
   | Except.error ex => throwError (ex.toMessageData (← getOptions))
 
+/-- Adds aliases to the current namespace, pointing at names that are defined in some other
+namespace.
```

Comment:
I think "alias" is misleading, because Std has an `alias` command and it means something different.

Reply:
Right, I think the new version addresses this issue, let me know what you think